### DISCUSSION TITLE
Did anyone got student developer pack?

### DISCUSSION THREAD

**Discussion by SirpiYugendar** [github/india#401] [Students]
I have been trying since 7th October, but I didn't receive an update. If anyone got the developer pack. Tell me what exactly you do to get?

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any `Official FRT Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Currently, we are helping all the `Future Ready Talent` Interns clear their doubts & queries on a personal basis either on chat or voice, whichever they prefer. We will also be assisting them on Projects & help them complete this internship 🤗

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

**Comment by TheSuranaverse** [comment]
@SirpiYugendar yes I have

**Reply by SirpiYugendar** [reply]
> @SirpiYugendar yes I have

how did you get the developer pack? can you guide me to get that?

**Reply by TheSuranaverse** [reply]
@SirpiYugendar join through the link: https://discord.gg/drQ7ySCdX2

Once joined, let me know pls. Will guide with all the process

**Reply by SirpiYugendar** [reply]
> @SirpiYugendar join through the link: https://discord.gg/drQ7ySCdX2
> 
> Once joined, let me know pls. Will guide with all the process

I joined.

**Reply by TheSuranaverse** [reply]
@SirpiYugendar your username there?

**Reply by SirpiYugendar** [reply]
> @SirpiYugendar your username there?

Username: sirpi005
